back/spec/models/notifications/cosponsor_of_your_idea_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Notifications::CosponsorOfYourIdea do
+  describe 'cosponsor_of_your_idea' do
+    it 'makes a notification on input_cosponsorship_accepted cosponsorship activity' do
+      idea = create(:idea, author: create(:user))
+      cosponsorship = create(:cosponsorship, idea: idea, status: 'accepted')
+      activity = create(
+        :activity,
+        item: cosponsorship,
+        user_id: cosponsorship.user_id,
+        action: 'input_cosponsorship_accepted'
+      )
+
+      notifications = described_class.make_notifications_on activity
```

Comment:
Detail, but probably a good idea to also check that there is only a single notification in the returned array, just in case.